guide/src/class.md

```diff
@@ -1239,8 +1239,8 @@ unsafe impl pyo3::type_object::PyTypeInfo for MyClass {
     #[inline]
     fn type_object_raw(py: pyo3::Python<'_>) -> *mut pyo3::ffi::PyTypeObject {
         <Self as pyo3::impl_::pyclass::PyClassImpl>::lazy_type_object()
-            .get_or_init(py)
-            .as_type_ptr()
+            .get_or_init_bound(py)
+            .as_ptr() as *mut pyo3::ffi::PyTypeObject
```

Comment:
I guess `as_type_ptr` here will exist after #3705 finally merges...

Reply:
Will fix this when that's merged.